zkvm/tests/zkvm.rs

```diff
@@ -79,3 +89,202 @@ fn issue() {
         }
     };
 }
+
+// questions:
+// - do we have to use separate nonce predicates for the two inputs (to ensure uniqueness if they're same qty?)
+// - is it enough (for testing) to get the value from the issue contract? (how do we create a "utxo" in one tx?)
```

Comment:
to fully test spending UTXOs, we might need two txs — one that issues the UTXOs and one that inputs them and spends them.

Reply:
That'd be a nicer test, but we can as well "make up" an `Input` like if it was already created and stored in the blockchain state machine. Since we don't have this state machine to police us, we can make up any UTXOs we want and pretend they were created by prior transactions.